rest_framework_gis/serializers.py

```diff
@@ -128,12 +131,15 @@ def to_representation(self, instance):
         # must be "Feature" according to GeoJSON spec
         feature["type"] = "Feature"
 
-        # required geometry attribute
-        # MUST be present in output according to GeoJSON spec
-        field = self.fields[self.Meta.geo_field]
-        geo_value = field.get_attribute(instance)
-        feature["geometry"] = field.to_representation(geo_value)
-        processed_fields.add(self.Meta.geo_field)
+        # geometry attribute
+        # must be present in output according to GeoJSON spec
+        if self.Meta.geo_field:
```

Comment:
```suggestion
        if self.Meta.geo_field is not None:
```

Reply:
I think this is good like this, we want to ignore stuff like "" or 0 that ended up here by accident.